experimental/packages/opentelemetry-instrumentation-grpc/src/grpc-js/index.ts

```diff
@@ -188,10 +189,15 @@ export class GrpcJsInstrumentation extends InstrumentationBase {
                   keys: carrier => Object.keys(carrier.getMap()),
                 }),
                 () => {
+                  const { service, method } = _extractMethodAndService(name)
+
                   const span = instrumentation.tracer
                     .startSpan(spanName, spanOptions)
                     .setAttributes({
                       [AttributeNames.GRPC_KIND]: spanOptions.kind,
```

Comment:
`AttributeNames.GRPC_KIND` recorded as `spanOptions.kind` seems like identical to `span.kind`. Do we really need this?

Reply:
Seems like it's exactly useless value